### PR TITLE
Removed unused code

### PR DESCRIPTION
**Pull request by ahcorde**

### CODE REVIEW

File: rviz_common/src/rviz_common/frame_manager.cpp [hunk 1]
```diff
@@ -291,13 +291,6 @@ const std::string & FrameManager::getFixedFrame()
...
 }
 
-std::string getTransformStatusName(const std::string & caller_id)
-{
-  std::stringstream ss;
-  ss << "Transform [sender=" << caller_id << "]";
-  return ss.str();
-}
-
 #if 0
```
Comment (clalancette) [inline rviz_common/src/rviz_common/frame_manager.cpp]: I'm also for removing this block, and basically any block that is `#if 0` out.  At this point in rviz2's life, if we need functionality like that we can:

1.  Get it from git history.
2.  Port it from ROS 1 rviz.
3.  Recreate it from scratch.

Comment (ahcorde) [reply rviz_common/src/rviz_common/frame_manager.cpp]: @clalancette  Removed all `#if 0` blocks

**Review by clalancette — APPROVED** [review]
Looks good to me with green CI.

**Comment by ahcorde** [comment]
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/19374/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/13901/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/20095/)

**Pull request merged**